src/RscChart.tsx

```diff
@@ -137,6 +146,8 @@ export const RscChart = forwardRef<ChartHandle, RscChartProps>(
 			// if the popover is closed, reset the selected data
 			if (!popoverIsOpen) {
```

Comment:
nit: this condition might be more intuitive if we swapped the statements.  Right now I read it like "if the popover isn't open (i.e. it closed)" ,  rather than my preferred reading of "if the popover just opened"